tests/project/tsconf.tests.ts

```diff
@@ -26,27 +26,25 @@ describe('TypeScript project config', () => {
         // Parse config from temp dir instead of root dir
         const cwd = await mkdtemp(join(tmpdir()));
         const project = join(rootDir, tsconfig, 'tsconfig.json');
-        const {code, stdout} = await tsc(['--project', project, '--showConfig'], {cwd});
 
-        expect(stdout).toStartWith('{');
-        expect(code).toBe(ExitStatus.Success);
-        const configJson = JSON.parse(stdout);
-        const config = ts.parseJsonConfigFileContent(configJson, ts.sys, project);
+        // Fire
+        const child = fork(tsc, ['--project', project, '--showConfig'], {silent: true, cwd});
 
+        const {response} = await childClosed(child, {serialization: 'json'});
+        expect(child.exitCode).toBe(ExitStatus.Success);
+        const config = ts.parseJsonConfigFileContent(response, ts.sys, project);
         expect(config.raw).toMatchSnapshot();
-    });
+    }, 100000);
 
-    // Slow and not executed automatically
+    // Slow test (skipped by default)
     it.skip.each(tsProjects)('should compile without errors: $tsconfig', async ({tsconfig}) => {
         // Compile config from temp dir instead of root dir
         const cwd = await mkdtemp(join(tmpdir()));
         const project = join(rootDir, tsconfig, 'tsconfig.json');
-        const {code, stdout} = await tsc(['--project', project, '--noEmit'], {cwd});
-        expect(stdout).toBeUndefined();
-        expect(code).toBe(ExitStatus.Success);
+
+        // Fire
+        const child = fork(tsc, ['--project', project, '--noEmit'], {silent: true, cwd});
```

Comment:
`--noEmit` should already be included in tsconfig.json now.

Reply:
I was thinking about this.. There are a few of those "compiler options" that should really be under a heading more along the lines of "output options". They follow different rules. Options like `noEmit` and `declarationOnly` and `outDir`, they may change depending on how you want to compile the code (say, rollup vs dev REPL, etc). 

So I thought it would be alright to "pin" this option. If for some reason you decided to ditch rollup and use tsc to handle the job, you would need to set noEmit to false, but I would still want it set this way here.